main.go

```diff
@@ -27,7 +27,7 @@ func initLogging() {
 		if (os.MkdirAll(flag.Lookup("log_dir").Value.String(), os.ModePerm) != nil) {
 			glog.Errorf("Failed to open custom log directory; defaulting to /tmp! Error: %v", flag.Lookup("log_dir").Value, err)
 		} else {
-			glog.Errorf("Failed to open custom log directory; attempting to create custom directory! Error: %v", flag.Lookup("log_dir").Value, err)
+			glog.V(3).Infof("Failed to open custom log directory; attempting to create custom directory! Error: %v", flag.Lookup("log_dir").Value, err)
```

Comment:
Heh, sorry for being so nit-picky, but on second thoughts, I think its best we not log anything at all in the case that the directory does not exist and we create it successfully. This is basically a routine task and should not be logged. If it is logged, it should be at debug level.

Reply:
Uhh I don't think glog for golang has dlog. You can specify a very high verbose level aka 8 or something

Reply:
If we don't create it successfully MkdirAll should return an error... which I didn't capture to use for the error log